doc/Language/classtut.pod6

```diff
@@ -192,69 +100,72 @@ X<|classes,has>
 X<|behavior>
 X<|classes,behavior>
 
-Raku, like many other languages, uses the C<class> keyword to define a
```

Comment:
We've already used "class" in the first example, so not a lot of sense in the original inception. Still, the same thing is kept so...